authorizer-app/src/app/services/auth.service.ts

```diff
@@ -0,0 +1,87 @@
+import { AuthData, AuthResponse, User } from '../models/auth.model';
+import { HttpClient, HttpHeaders, HttpParams } from '@angular/common/http';
+
+import { Injectable } from '@angular/core';
+import { environment } from 'src/environments/environment';
+import { map } from 'rxjs/operators';
+import { of } from 'rxjs';
+import { storageItems } from '../enums/storage';
+
+@Injectable()
+export class AuthService {
```

Comment:
Can we create an interface called `AuthService` and then this will implement that and perhaps be called `ManagementPortalAuthService` or if more generic then `AuthorizationCodeAuthService`. This will make it easy to extend in the future and since it is already injected we can provide whichever AuthService we like.

Reply:
Okay sure 👍